tuta-sdk/rust/src/util/mod.rs

```diff
@@ -1,6 +1,18 @@
+use base64::alphabet::Alphabet;
+use base64::engine::GeneralPurpose;
+
 #[cfg(test)]
 pub mod test_utils;
 
+const BASE64EXT_ALPHABET: Alphabet = match Alphabet::new("-0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz") {
+    Ok(x) => x,
+    Err(_) => panic!("creation of alphabet failed")
+};
+
+pub const BASE64_EXT: GeneralPurpose = GeneralPurpose::new(
```

Comment:
we should transfer a comment from ts code about what it is

Reply:
Comment added.